src/test/java/redis/clients/jedis/tests/JedisSentinelPoolTest.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Most of my comments in `JedisPoolTest.java` are also applicable in this class. So use those suggestions :)

Reply:
done